packages/jsActions/native-mobile-resources/config/rollup.config.js

```diff
@@ -0,0 +1,114 @@
+import { red, yellow } from "colors";
+import { mkdirSync, promises } from "fs";
+import { join, relative } from "path";
+import clear from "rollup-plugin-clear";
+import command from "rollup-plugin-command";
+import { cp } from "shelljs";
+import { nodeResolve } from "@rollup/plugin-node-resolve";
+import typescript from "@rollup/plugin-typescript";
+import { collectNativeDependencies } from "../../../tools/pluggable-widgets-tools/configs/rollup-plugin-collect-native-dependencies";
+
+const cwd = process.cwd();
+
+export default async args => {
+    const jsActionTargetFolder = `javascriptsource/${args.configProject ?? "nativemobileresources"}/actions`;
+    const result = [];
+    const files = await findFiles();
+    const outDir = join(cwd, "dist");
+
+    files.forEach((file, i) => {
+        const fileInput = file.path.replace(join(cwd, "/"), "");
```

Comment:
Does this `join` actually do anything?

Reply:
(Isn't this a case for `path.relative` actually? 🤔)

Reply:
Yes, just changed